sw/device/lib/dif/dif_lc_ctrl.h

```diff
@@ -208,6 +208,22 @@ typedef enum dif_lc_ctrl_id_state {
   kDifLcCtrlIdStateInvalid,
 } dif_lc_ctrl_id_state_t;
 
+/**
+ * A lifecycle controller mutex acquire and release code.
+ */
+typedef enum dif_lc_ctrl_mutex_code {
+  /**
+   * According to LC_CTRL register spec [hw/ip/lc_ctrl/data/lc_ctrl.hjson],
```

Comment:
Hi Miguel @moidx  sorry I did not find any examples for referring to HW spec. Could you confirm if this is the correct syntax for SW?

Reply:
You can add a link directly to the generated docs like it is done here: https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/lib/dif/dif_csrng.c;l=21

Reply:
Thanks for the reference. I will follow that.